Sources/StreamChatUI/CommonViews/ContainerStackView.swift

```diff
@@ -222,18 +266,18 @@ public class ContainerStackView: UIView {
             }
         }
 
+        // Add constraints for the layout alignment.
         subviews.forEach { subview in
-
             if axis == .horizontal {
-                if alignment == .axisLeading || alignment == .fill {
+                if alignment == .leading || alignment == .top || alignment == .fill {
```

Comment:
This is not really needed, `top` has the same raw value as `leading`. It's just syntax sugar :)